packages/lib/services/action.ts

```diff
@@ -0,0 +1,67 @@
+"use server";
```

Comment:
Do you know if `"use server" throws a build error when imported into a client component?
you can also additionally import:
```
import "server-only";
```
to make sure a build error is thrown when this file is imported into a client component.

Reply:
that's the part of the docs with the server-only package that I refer to:
https://nextjs.org/docs/app/building-your-application/data-fetching/caching#the-server-only-package